app/validation/validator.py

```diff
@@ -156,12 +191,20 @@ def _validate_variants(self, block, answer_ids_with_group_id, numeric_answer_ran
         if len(question_variants) == 1 or len(content_variants) == 1:
             errors.append(self._error_message('Variants contains fewer than two variants - block: {}'.format(block['id'])))
 
+        number_of_answers = set()
+
         for variant in question_variants:
             errors.extend(self._validate_questions(variant, numeric_answer_ranges))
+            number_of_answers.add(len(variant['question']['answers']))
+
+        if len(number_of_answers) > 1:
+            errors.append(self._error_message('Variants in block: {} contain different numbers of answers'.format(block['id'])))
```

Comment:
It wold make more sense for this validation to be in `_ensure_all_variant_ids_consistent`

Reply:
Done